packages/taskrunner/src/main/java/software/uncharted/terarium/taskrunner/models/task/TaskResponse.java

```diff
@@ -15,6 +15,8 @@ public class TaskResponse implements Serializable {
 	private String script;
 	private TaskStatus status;
 	private byte[] output;
+	private Object type;
```

Comment:
Should this be `ClientEventType` not `Object`?

Reply:
This is taskrunner package and I wasn't sure if it's possible to import types/classes from the hmi server. So I set it to Object so can it can handle arbitrary object when the task messages get encoded/decoded.

Reply:
We can't import the type directly from hmi-server since it causes config issues with spring. I recall when i tried to import TaskRequest / TaskResponse from hmiserver it imported the entire app config.

Reply:
I don't think we need an extra `"type"` fiield on the TaskRequewst / TaskResponse. The `script` field can be used as a `"type"`. We can either use it directly, or map from the `String` to the `ClientEventType` on the hmi-server.

Reply:
Done